app/views/favorites/_favorite.html.erb

```diff
@@ -0,0 +1,34 @@
+<tr>
+  <td><%= favorite.book.name %></td>
```

Comment:
chú tìm hiểu về delegate nhé
https://viblo.asia/p/delegate-in-rails-WEMGBVVAGQK